modules/ROOT/pages/am-installing.adoc

```diff
@@ -114,7 +127,9 @@ Ensure that the server has the minimum hardware requirements for Anypoint Monito
 . Meet the prerequisites in <<prereqs>>:
 +
 Mule runtime engine must be installed and running.
-. Follow the steps in xref:monitoring-settings-page.adoc[] to download the agent's `am.zip` file and install the agent from *Settings*, in Anypoint Monitoring.
+. Follow the steps in xref:monitoring-settings-page.adoc[] to download the _latest version_ of the agent's `am.zip` file and install the agent from *Settings*, in Anypoint Monitoring.
++
+To install an _earlier_ version, such as a 2.x version, you must download the agent's zip file from the *Anypoint Monitoring* tab in the https://help.mulesoft.com/s/downloads?tabset-169e6=38694[download portal]. 
```

Comment:
TBD: 130-132 above currently assume we get the 2.x agent up on that portal and the 3.x is available from Settings.